### PR TITLE
chore: run cargo update and fixes

### PR DESCRIPTION
**Pull request by tiagolobocastro**

### CODE REVIEW

**Comment by tiagolobocastro** [comment]
bors try

File: control-plane/rest/service/src/v0/swagger_ui.rs [hunk 1]
```diff
@@ -34,10 +34,17 @@ fn get_swagger_html(spec_uri: &str) -> Result<String, String> {
 #[derive(Clone)]
 struct GetSwaggerUi(Result<String, String>);
 
-impl Handler<(), Ready<Result<HttpResponse, Error>>> for GetSwaggerUi {
+impl Handler<()> for GetSwaggerUi {
+    type Output = Result<HttpResponse, Error>;
+    type Future = Ready<Result<HttpResponse, Error>>;
+
     fn call(&self, _: ()) -> Ready<Result<HttpResponse, Error>> {
```
Comment (paulyoong) [inline control-plane/rest/service/src/v0/swagger_ui.rs:41]: You can now change the return type to `Ready<Self::Output>`

**Review by paulyoong — APPROVED** [review]

File: control-plane/rest/service/src/v0/swagger_ui.rs (hunk 1 above)
Comment (tiagolobocastro) [reply control-plane/rest/service/src/v0/swagger_ui.rs:41]: Ah yes, Future can be that and then the result can just be Self::Future

**Comment by tiagolobocastro** [comment]
bors try

**Comment by tiagolobocastro** [comment]
bors merge

**Pull request merged**